packages/core/upload/admin/src/components/EditFolderDialog/EditFolderDialog.js

```diff
@@ -229,54 +222,54 @@ export const EditFolderDialog = ({ onClose, folder, parentFolderId }) => {
                     </Stack>
                   </GridItem>
                 </Grid>
+              </ModalBody>
 
-                <VisuallyHidden>
-                  <button type="submit" tabIndex={-1} ref={submitButtonRef} name="hidden-submit">
-                    {formatMessage({ id: 'submit', defaultMessage: 'Submit' })}
-                  </button>
-                </VisuallyHidden>
-              </Form>
-            )}
-          </Formik>
-        </ModalBody>
-
-        <ModalFooter
-          startActions={
-            <Button onClick={() => onClose()} variant="tertiary" name="cancel">
-              {formatMessage({ id: 'cancel', defaultMessage: 'Cancel' })}
-            </Button>
-          }
-          endActions={
-            <Stack horizontal spacing={2}>
-              {isEditing && canUpdate && (
-                <Button
-                  type="button"
-                  variant="danger-light"
-                  onClick={() => setShowConfirmDialog(true)}
-                  name="delete"
-                >
-                  {formatMessage({
-                    id: 'modal.folder.create.delete',
-                    defaultMessage: 'Delete folder',
-                  })}
-                </Button>
-              )}
+              <ModalFooter
+                startActions={
+                  <Button
+                    onClick={() => onClose()}
```

Comment:
In case `!isEditFolderLoading && onClose()` makes sense (which I don't think) it would need to be added here too.

Reply:
Actually the user can't access the onClick event because the Save button is in loading state, and the Cancel / Delete buttons are on disabled state

<img width="560" alt="image" src="https://user-images.githubusercontent.com/71838159/173526550-8040cbe1-a256-4055-b0c4-d4d1319f359d.png">